fenix/app/src/main/java/org/mozilla/fenix/compose/DropdownMenu.kt

```diff
@@ -21,21 +27,24 @@ import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.DpOffset
 import androidx.compose.ui.unit.dp
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.PrimaryButton
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
- * Popup action dropdown menu.
+ * Root popup action dropdown menu.
  *
  * @param menuItems List of items to be displayed in the menu.
  * @param showMenu Whether or not the menu is currently displayed to the user.
+ * @param cornerShape Shape to apply to the corners of the dropdown.
  * @param onDismissRequest Invoked when user dismisses the menu or on orientation changes.
  * @param modifier Modifier to be applied to the menu.
  * @param offset Offset to be added to the position of the menu.
  */
 @Composable
-fun DropdownMenu(
+private fun DropdownMenu(
```

Comment:
I wonder if this naming will overlap with the actual Dropdown Menu component in figma?

Reply:
hrmm, good point. Maybe just `Menu`?

Reply:
`Menu` makes sense given it's the menu component in the figma. I would also suggest renaming the file name as well to match.

Reply:
good catch on the filename